### ISSUE TITLE
Changes in commit 4f4fd10 lead to more slashes getting stripped, than neccessary

### ISSUE THREAD

**Issue by Karmel0x** [smalot/pdfparser#322]
Sample pdf: [cv3.pdf](https://github.com/smalot/pdfparser/files/5026842/cv3.pdf)

Where is problem:
![img3](https://user-images.githubusercontent.com/25367564/89385411-d95d5b00-d6ff-11ea-8d8b-e7d638b2e5a4.PNG)

Latest pdfparser commit output:
![img1](https://user-images.githubusercontent.com/25367564/89385078-5cca7c80-d6ff-11ea-9e96-ec080c0d3e3b.PNG)

Output with reversed changes from commit 4f4fd10:
![img2](https://user-images.githubusercontent.com/25367564/89385083-5f2cd680-d6ff-11ea-80ce-5c57f8cf5f73.PNG)

**Comment by k00ni** [comment]
Hi @Karmel0x, so if I understand it correctly commit 4f4fd10 made things worse? Based on the code more slashes get stripped than before. Can someone else approve this observation?

/CC @machekhin

**Comment by Karmel0x** [comment]
It seems so, at least in my case.

**Comment by j0k3r** [comment]
Should we revert the commit and hope to find a better solution?

**Comment by k00ni** [comment]
Can anyone write a simple test which proofs this observation (take the posted PDF from first post and check output)? I am :+1 to revert, if it made things worse.

**Comment by Connum** [comment]
It would be great if the author of #260, @OneSoftIO, could provide a dummy PDF that replicates the issue from before the fix in 4f4fd10. Because when we revert the commit, we'll re-introduce the other bug again. An optimal test case would test both PDFs for correct parsing and help find a solution that fixes both issues at the same time.

**Comment by Connum** [comment]
Here's a test for `ParserTest.php` (saving the PDF from the opening post as `samples/bugs/Issue322.pdf`:

```php
    /**
     * Test that issue related pdf can now be parsed
     *
     * @see https://github.com/smalot/pdfparser/issues/322
     */
    public function testIssue322()
    {
        $filename = $this->rootDir.'/samples/bugs/Issue322.pdf';

        $document = $this->fixture->parseFile($filename);

        $this->assertStringContainsString('this text isn’t working properly, I’ve edited it in Google Documents', $document->getText());
    }
```

This will fail in the current master state, but pass with the changes from `4f4fd10` reverted.
However, as I said, we'd need a sample for #260 as well...
